src/core/Akka.Cluster/Cluster.cs

```diff
@@ -62,13 +62,29 @@ public static Cluster Get(ActorSystem system)
             return system.WithExtension<Cluster, ClusterExtension>();
         }
 
+        static Cluster()
+        {
+            bool GetAssertInvariants()
```

Comment:
Enabled assert invariants inside the `Cluster` when the `"akka.cluster.assert"` environment variable is set